lib/Service/RelatedService.php

```diff
@@ -451,14 +451,14 @@ private function filterUnavailableResults(array $result): array {
 	}
 
 	/**
-	 * @param IRelatedResource[] $paths
+	 * @param IRelatedResource $current
```

Comment:
```suggestion
```
Same as above

Reply:
we already had a discussion about this, we keep phpdoc